Tests/KSCrashRecordingTests/KSCrashDoctor_Tests.m

```diff
@@ -0,0 +1,30 @@
+#import <XCTest/XCTest.h>
+
+#import "KSCrashDoctor.h"
+
+// This is duplicated from `KSCrashReportFixer_Tests.m`
```

Comment:
Would it be possible to move this code to the `KSCrashTestTools` module to avoid duplication? I suggest we could create a separate `.h` file named `KSTestModuleConfig.h` in `Sources/KSCrashTestTools/include/` for better organization. Also, don’t forget to add an include guard to the header file. What do you think?

Reply:
I'm not too sure why but I'm not able to get `SWIFTPM_MODULE_BUNDLE` to be defined in this manner. There's some magic going on with this I'm not getting.

Reply:
Oh, I see. Maybe this macro is only defined for `.target' with `resources:`. I guess we've got nothing left but duplication or definition on each test-target-level. @bamx23 any thoughts on this?

Reply:
It works in a .bundle but not a .xctest which is where it ends up in https://github.com/kstenerud/KSCrash/pull/472/commits/975bbfd4c3fe8f65c7d90cc3d3b8155b20fc2aaf

Reply:
OK. What if we try to create the same header, but in `Tests/KSCrashRecordingTests`?

Reply:
That did it, good call!

Reply:
I had the same issue that I can't use `SWIFTPM_MODULE_BUNDLE` in another common header.